### ISSUE TITLE
How to retrieve key or mouse input?

### ISSUE THREAD

**Issue by gsfordham** [autokey/autokey#85]
Basically, this is just a question.

I want to create an autocorrect script that will manage several tens or a few hundred different "abbreviations". What is the variable that is used to give me whatever I just entered before triggering the script?

For example, if I type "eror", and I want it to correct to "error", how do I actually select it from one script, where "eror" is one of many triggers?

I want something like:
```python
if re.search(r"eror", input):
    keyboard.send_keys("error")
elif re.search("suppsoe", input):
    keyboard.send_keys("suppose")
elif [...]
```
I was looking under the docs, but I can't find exactly how to access any of the string buffers that are being used.
So, I have no idea what the variable "input" would be.

TL;DR: Under what variable is the abbreviation stored, so I can pass the last-sent abbreviation to a single script and process based on that? I need to know, IN the script, WHICH trigger caused the script to execute.

I would also be using this for a bunch of expansions, as well, but I don't want several hundred autocorrect files. I did this on AutoHotKey on Windows.

**Comment by troxor** [comment]
Please do not request support on the issue tracker. Instead, head over to the [autokey-users mailing list](https://groups.google.com/forum/#!forum/autokey-users), ask on IRC (#autokey on Freenode) or [Gitter](https://gitter.im/autokey-py3).

We'd appreciate it if you take a look at the [Problem reporting guide](https://github.com/autokey/autokey/wiki/Problem-Reporting-Guide) before posting. By providing as much information as you can, you'll have a much better chance of getting a good answer in less time.
